### DISCUSSION TITLE
kernel environment tidak terbaca dipython

### DISCUSSION THREAD

**Discussion by yulekristina** [teamalgoritma/community#188] [Q&A] [answered]
<img width="531" alt="image" src="https://user-images.githubusercontent.com/107384192/178137320-507f21f8-baf6-4b9a-bdd1-5c126cb4e67f.png">
 Ada yang tau setiap mo run dengan jalanin pke kernel (environment baru) error pdhal klo djalanin pke kernel python bisa..
ada yang tau solusinya ngga ya? terima kasih sebelumnya

**Comment by triarm** [comment, marked as answer]
Hi @yulekristina ,
Terkait error tersebut, apakah Anda sudah menjalankan cell sebelumnya untuk membaca data playstore? Karena dilihat dari errornya sepertinya Anda belum mendefinisikan object playstore tersebut.

Demikian yang dapat kami sampaikan. Semoga menjawab.
